examples/storage/cassandra/README.md

```diff
@@ -246,7 +246,7 @@ spec:
           timeoutSeconds: 5
         # These volume mounts are persistent. They are like inline claims,
         # but not exactly because the names need to match exactly one of
-        # the pet volumes.
+        # the pod volumes.
```

Comment:
This part (from `<!-- BEGIN MUNGE -->` to `<!-- END MUNGE -->`) is auto-generated, so you'll need to modify `cassandra-statefulset.yaml` and then run `hack/update-munge-docs.sh` to update it